src/de_main.c

```diff
@@ -416,7 +416,12 @@ static void Game_Iterate(AppState *app)
     bool run_simulation = (!app->debug.pause_on_every_frame || !app->debug.paused_frame);
     if (run_simulation)
     {
-        Game_AdvanceSimulation(app);
+        float time_step = 0.25;
+        while (app->dt > 0)
```

Comment:
```suggestion
        while (app->dt >= time_step)
```
This would be more correct maybe... depends how you think about it,
But also has an advantage of keeping the accumulator variable always positive and it will be easier to calculate how many ticks are left. If we ever want to query that information it will be just a simple division (`app->dt / time_step`).